plugin/storage/integration/es_index_cleaner_test.go

```diff
@@ -60,7 +60,7 @@ func TestIndexCleaner_doNotFailOnEmptyStorage(t *testing.T) {
 }
 
 func TestIndexCleaner_doNotFailOnFullStorage(t *testing.T) {
-	skipUnlessEnv(t, "elasticsearch", "opensearch")
+	SkipUnlessEnv(t, "elasticsearch", "opensearch")
```

Comment:
this little change is a good candidate for a separate PR. It would significantly reduce the number of files in this PR, making it more focused.

Reply:
I see, I'll do that